### PR TITLE
Display the list of habits in profile

### PR DESCRIPTION
**Pull request by Enrico-Benedettini**

### CODE REVIEW

File: app/src/main/java/com/github/mateo762/myapplication/profile/ProfileActivity.kt
Comment (AnirudhhRamesh) [inline app/src/main/java/com/github/mateo762/myapplication/profile/ProfileActivity.kt:1]: Remove commented out code
Good and clean use of the database ref
l.116 and l.118 haven't seen this type of ScrollView and ParentView setting, is this safe/good practice?

File: app/src/main/java/com/github/mateo762/myapplication/upload_gallery/UploadPictureActivity.kt
Comment (AnirudhhRamesh) [inline app/src/main/java/com/github/mateo762/myapplication/upload_gallery/UploadPictureActivity.kt:1]: Remove commented out code if not needed

**Review by AnirudhhRamesh — APPROVED** [review]
Overall very good code, particularly like the way you access the database
Some areas for improvement:
- Make sure to remove commented out code
- Haven't seen the use of ScrollView(this@ProfileActivity) or LinearLayout(this@ProfileActivity), is this good practice?

File: app/src/main/java/com/github/mateo762/myapplication/profile/ProfileActivity.kt
Comment (AnirudhhRamesh) [reply app/src/main/java/com/github/mateo762/myapplication/profile/ProfileActivity.kt:1]: Good code commenting :))

**Review by AnirudhhRamesh — APPROVED** [review]
Great job, good that you documented your code

File: app/src/main/java/com/github/mateo762/myapplication/profile/ProfileActivity.kt
Comment (Enrico-Benedettini) [reply app/src/main/java/com/github/mateo762/myapplication/profile/ProfileActivity.kt:1]: Yes the ScrollView is particularly needed in this case, as we're displaying some images in an horizontal scroll and the rest is instead supposed to be scrolled vertically. The commented code was just a draft, is now removed

**Pull request merged**